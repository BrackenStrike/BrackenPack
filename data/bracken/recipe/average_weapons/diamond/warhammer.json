{
	"type": "minecraft:smithing_transform",
	"template": "minecraft:ward_armor_trim_smithing_template",
	"base": "minecraft:diamond_sword",
	"addition": "minecraft:diamond",
	"result": {
		"id": "minecraft:diamond_sword",
		"components": {
			"minecraft:item_name": "Diamond Warhammer",
			"minecraft:item_model": "bracken:average_weapons/diamond/warhammer",
			"minecraft:lore": [
				{"translate":"","color":"white","italic":false},
				{"translate":"When in Main Hand:","color":"gray","italic":false},
				{"translate": "7 Attack Damage","color":"dark_green","italic":false},
				{"translate": "1.6 Attack Speed","color":"dark_green","italic":false},
				{"translate": "20% Knockback Resistance","color":"dark_green","italic":false}
			],
			"minecraft:attribute_modifiers":[{
					"type": "minecraft:attack_damage",
					"id": "bracken:diamond_warhammer",
					"amount": 6,
					"operation": "add_value",
					"slot": "mainhand"
				},{
					"type": "minecraft:attack_speed",
					"id": "bracken:diamond_warhammer",
					"amount": -2.4,
					"operation": "add_value",
					"slot": "mainhand"
				},{
					"type": "minecraft:knockback_resistance",
					"id": "bracken:diamond_warhammer",
					"amount": 0.2,
					"operation": "add_value",
					"slot": "mainhand"
				}],
				"minecraft:tooltip_display": {
        "hidden_components": [
          "minecraft:attribute_modifiers"
        ]
      }
		}
	}
}