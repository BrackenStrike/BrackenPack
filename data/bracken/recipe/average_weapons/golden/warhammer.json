{
	"type": "minecraft:smithing_transform",
	"template": "minecraft:ward_armor_trim_smithing_template",
	"base": "minecraft:golden_sword",
	"addition": "minecraft:gold_ingot",
	"result": {
		"id": "minecraft:golden_sword",
		"components": {
			"minecraft:item_name": "\"Golden Warhammer\"",
			"minecraft:item_model": "bracken:average_weapons/golden/warhammer",
			"minecraft:lore": [
				"{\"translate\":\"\",\"color\":white,\"italic\":false}",
				"{\"translate\":\"When in Main Hand:\",\"color\":gray,\"italic\":false}",
				"{\"translate\":\" 4 Attack Damage\",\"color\":dark_green,\"italic\":false}",
				"{\"translate\":\" 1.6 Attack Speed\",\"color\":dark_green,\"italic\":false}",
				"{\"translate\":\" 20% Knockback Resistance\",\"color\":dark_green,\"italic\":false}"
			],
			"minecraft:attribute_modifiers": {
				"modifiers": [{
					"type": "minecraft:attack_damage",
					"id": "bracken:golden_warhammer",
					"amount": 3,
					"operation": "add_value",
					"slot": "mainhand"
				},{
					"type": "minecraft:attack_speed",
					"id": "bracken:golden_warhammer",
					"amount": -2.4,
					"operation": "add_value",
					"slot": "mainhand"
				},{
					"type": "minecraft:knockback_resistance",
					"id": "bracken:golden_warhammer",
					"amount": 0.2,
					"operation": "add_value",
					"slot": "mainhand"
				}],
				"show_in_tooltip": false
			}
		}
	}
}